bn-models/src/models/wcgop/wcgop-operation.ts

```diff
@@ -17,58 +55,185 @@ declare type ProductDeliveryState = any; // TODO Lookup
 
 // TODO: Create both WCGOP Trawl and Fixed Gear haul types?
 export interface WcgopOperation extends BaseOperation {
-  biolist?: number;
-  catches?: WcgopCatch[];
-  operationNum?: number; // Sequential
-  locations?: FishingLocation[];
+  /*
+  Attributes from extended interfaces are accessible in this interface.
+
+  WcgopOperattion <-- BaseOperation <-- Base
```

Comment:
same as the comment above, believe we don't need a list of base attributes here.